sympy/vector/coordsysrect.py

```diff
@@ -173,6 +174,61 @@ def __str__(self, printer=None):
     def __iter__(self):
         return iter([self.i, self.j, self.k])
 
+    def connect_to_standard_cartesian(self,  curv_coord_type):
```

Comment:
Well, it's true. 
It could be easier for us if we'd have explicit information about type of argument, like 
`trans_eq=...` or `coord_type=...`